plugins/woocommerce/src/Blocks/BlockTypes/PaymentMethodIcons.php

```diff
@@ -0,0 +1,221 @@
+<?php
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use WP_Block;
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * PaymentMethodIcons class.
+ */
+class PaymentMethodIcons extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'payment-method-icons';
+
+	/**
+	 * Get the frontend script handle for this block type.
+	 *
+	 * @param string $key Data to get, or default to everything.
+	 * @return array|string
+	 */
+	protected function get_block_type_script( $key = null ) {
+		return null;
+	}
+
+	/**
+	 * Get the frontend style handle for this block type.
+	 *
+	 * @return string[]
+	 */
+	protected function get_block_type_style() {
+		return array_merge( parent::get_block_type_style(), [ 'wc-blocks-packages-style' ] );
+	}
+
+	/**
+	 * Extra data passed through from server to client for block.
+	 *
+	 * @param array $attributes  Any attributes that currently are available from the block.
+	 */
+	protected function enqueue_data( array $attributes = [] ) {
+		parent::enqueue_data( $attributes );
+		$this->asset_data_registry->add( 'availablePaymentMethods', $this->get_available_payment_methods() );
+	}
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content    Block content.
+	 * @param WP_Block $block      Block instance.
+	 * @return string Rendered block type output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		$enabled_card_types    = $this->get_enabled_card_types();
+		$other_payment_methods = $this->get_other_payment_method_icons();
+
+		if ( empty( $enabled_card_types ) && empty( $other_payment_methods ) ) {
+			return '';
+		}
```

Comment:
Super minor but seems it could be replaced with `get_available_payment_methods` for simplicity

```suggestion
		$payment_methods = $get_available_payment_methods();

		if ( empty( $payment_methods ) ) {
			return '';
		}
```

Reply:
Yes, thank you! I missed this when refactoring.